src/cmd/api/controllers/video_status_handler.go

```diff
@@ -0,0 +1,64 @@
+package controllers
+
+import (
+	"database/sql"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/db/dao"
+)
+
+type VideoStatus struct {
+	Status string `json:"status" example:"UPLOADED"`
+}
+
+type VideoStatusHandler struct {
+	MariadbClient *sql.DB
+}
+
+// @Summary Get video status
+// @Description Get video status
+// @Tags status
+// @Accept plain
+// @Produce plain
+// @Param id path string true "Video ID"
+// @Success 200 {Json} Json status:"Video status"
+// @Failure 400 {object} object
+// @Failure 500 {object} object
+// @Router api/v1/videos/{id}/status [get]
+func (v VideoStatusHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	log.Debug("GET VideoStatusHandler - parameters ", vars)
+
+	id, exist := vars["id"]
```

Comment:
It can be applied at the other place too but if the ID is a uuid, we must verify that it's a valid one to protect us from malicious code injection.
Bonus question, why, do we use VARCHAR for the id and not a UUID type ? (https://mariadb.com/kb/en/uuid-data-type/)